jwst/associations/association.py

```diff
@@ -265,7 +269,8 @@ def load(
 
         Returns
         -------
-        The Association object
+        association
```

Comment:
The data type should be specified, though admittedly that would be redundant.  What was wrong with "The Association object"?